### PR TITLE
Add a lookup data checker

### PR DESCRIPTION
**Pull request by ipetr0v**
This PR adds an utility that checks the correctness of the weather lookup data file.

### CODE REVIEW

File: oak_functions/lookup_data_checker/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Lookup Data Checker
+
+The binary in this directory checks the correctness of the lookup data file.
```
Comment (conradgrobler) [inline oak_functions/lookup_data_checker/README.md:3]: perhaps add a code example on how to use it?

**Review by conradgrobler — APPROVED** [review]
Nice, thanks!

File: oak_functions/lookup_data_checker/README.md (hunk 1 above)
Comment (ipetr0v) [reply oak_functions/lookup_data_checker/README.md:3]: Done

**Review by tiziano88 — APPROVED** [review]
Nice! ideally this could be also a utility library that runs on the output from the generator, and there could be an integration test between the two. Perhaps create an issue to track that @ipetr0v ? (does not have to be done now, we can just leave it around)

**Comment by ipetr0v** [comment]
Created https://github.com/project-oak/oak/issues/2578

**Pull request merged**